_config.yml

```diff
@@ -97,91 +97,55 @@ plugins:
 # Site configuration for the Jekyll 3 Pagination Gem
 # The values here represent the defaults if nothing is set
 pagination:
-
   # Site-wide kill switch, disabled here it doesn't run at all
   enabled: true
-
   # Set to 'true' to enable pagination debugging. This can be enabled in the site config or only for individual pagination pages
   debug: false
-
   # The default document collection to paginate if nothing is specified ('posts' is default)
   collection: 'posts'
-
   # How many objects per paginated page, used to be `paginate` (default: 0, means all)
   per_page: 3
-
   # The permalink structure for the paginated pages (this can be any level deep)
   permalink: '/page/:num/' # Pages are index.html inside this folder (default)
   #permalink: '/page/:num.html' # Pages are simple html files
   #permalink: '/page/:num' # Pages are html files, linked jekyll extensionless permalink style.
-
   # Optional the title format for the paginated pages (supports :title for original page title, :num for pagination page number, :max for total number of pages)
   title: ':title - page :num'
-
   # Limit how many pagenated pages to create (default: 0, means all)
   limit: 0
-
   # Optional, defines the field that the posts should be sorted on (omit to default to 'date')
   sort_field: 'date'
-
   # Optional, sorts the posts in reverse order (omit to default decending or sort_reverse: true)
   sort_reverse: true
-
   # Optional, the default category to use, omit or just leave this as 'posts' to get a backwards-compatible behavior (all posts)
   category: 'posts'
-
   # Optional, the default tag to use, omit to disable
   tag: ''
-
   # Optional, the default locale to use, omit to disable (depends on a field 'locale' to be specified in the posts,
   # in reality this can be any value, suggested are the Microsoft locale-codes (e.g. en_US, en_GB) or simply the ISO-639 language code )
   locale: ''
-
  # Optional,omit or set both before and after to zero to disable.
  # Controls how the pagination trail for the paginated pages look like.
   trail:
     before: 2
     after: 2
-
   # Optional, the default file extension for generated pages (e.g html, json, xml).
   # Internally this is set to html by default
   extension: html
-
   # Optional, the default name of the index file for generated pages (e.g. 'index.html')
   # Without file extension
   indexpage: 'index.html'
 
 
+assets:
+  sources:
+    - assets/css
+    - assets/fonts
+    - assets/images
+    - assets/js
+    - assets/videos
```

Comment:
I think all of these sources are included in the defaults: https://github.com/envygeeks/jekyll-assets#configuration

Reply:
good call. removed.